MANIFEST.in

```diff
@@ -0,0 +1,15 @@
+include pyproject.toml
+
+# Include the README
+include *.md
+
+# Include the license file
+include LICENSE.txt
+
+# Include setup.py
+include setup.py
+
+# Include the data files
+recursive-include data *include versioneer.py
+include yeat/_version.py
+include versioneer.py
```

Comment:
May want to check if the Snakefile is correctly copied without explicitly listing it here.